.gitignore

```diff
@@ -24,5 +24,7 @@ _testmain.go
 *.prof
 
 release/
+vendor/
```

Comment:
Is this a good idea? Won't it be possible that builds will no longer be identically reproducible if the source is removed?

Reply:
The deps will be identical because https://github.com/drone-plugins/drone-downstream/pull/42/files#diff-bd247e83efc3c45ae9e8c47233249f18 defines the exact commit of the dependency and beside that we can easily accept pull requests that update or add dependencies without reviewing the whole vendor directory changes.

Reply:
Yes it does, however if the source (on github) is removed then they will not be?
I'm fine with the change, just wanted to check.

Reply:
I think we are all aware of that, but we should anyway pick the deps carefully and if the lib gets deleted we need to find an alternative anyway :)